lib/curly/component_compiler.rb

```diff
@@ -64,8 +64,7 @@ def append_keyword_arguments(code)
     end
 
     def invalid_signature?
-      positional_params = param_types.select {|type| [:req, :opt].include?(type) }
-      positional_params.size > 1
+      param_types.any? { |type| [:req, :opt].include?(type) }
```

Comment:
That specific change changes nothing; what was there before does the exact same thing as what is there now, except what is there now is more succinct (and faster).  I'm not sure what you're trying to say, though.